### PR TITLE
Add contact us section to homepage

### PR DESCRIPTION
**Pull request by minhngo3818**
Add "Contact Us" section to homepage. Responsive feature was included.

Solved #54 

Demo 
Screenshot 2022-09-05 230256

### CODE REVIEW

File: client/src/pages/home/components/contact-us/ContactUs.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+				</div>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+					eiusmod tempor
+					<a id="form" href="#google-form">
+						Registration form
+					</a>
+				</p>
+				<p>
+					We care about your oppinion!
```
Comment (KHVBui) [inline client/src/pages/home/components/contact-us/ContactUs.js:56]: typo on "oppinion"

File: client/src/pages/home/components/contact-us/contact-us.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+@media only screen and (max-width: 548px) {
+	.contact-header {
+		margin: 0.5rem auto;
+	}
+	.contact-header h1 {
+		font-size: 30px;
+		text-align: center;
+		margin: auto;
+	}
+}
```
Comment (KHVBui) [inline client/src/pages/home/components/contact-us/contact-us.css:115]: Looks too indented on the left on smaller screen size.

Recommend adjusting padding like the About Us section.

Screen Shot 2022-09-05 at 11 41 23 PM

**Review by KHVBui — CHANGES_REQUESTED** [review]
We should also probably have a conversation about design choices later because "About Us" and "Contact Us" aren't consistent stylistically right now. We don't need to do that for this PR though.

**Review by KHVBui — APPROVED** [review]
Good to go!

**Comment by minhngo3818** [comment]
I just added more space between picture and text.

**Review by KHVBui — APPROVED** [review]

**Pull request merged**